dev/com.ibm.ws.webcontainer.servlet.3.1_fat/fat/src/com/ibm/ws/webcontainer/servlet31/fat/FATSuite.java

```diff
@@ -85,8 +81,8 @@ public static void generateHelpFile() {
     @ClassRule
     public static RepeatTests r = RepeatTests
                     .with(new EmptyAction().fullFATOnly())
-                    .andWith(FeatureReplacementAction.EE8_FEATURES()
-                                    .fullFATOnly())
-                    .andWith(FeatureReplacementAction.EE9_FEATURES()); 
-                    
+                    .andWith(FeatureReplacementAction.EE8_FEATURES().fullFATOnly())
+                    .andWith(FeatureReplacementAction.EE9_FEATURES().fullFATOnly())
+                    .andWith(FeatureReplacementAction.EE10_FEATURES());
+
```

Comment:
should EE10_FEATURES() also be fullFATOnly() ?

Reply:
No, Paul's code looks right. Only the EE10_FEATURES features would run in lite mode. When full mode rules, then the the original, ee8, ee9, and ee10 versions would all run.  

For the example, the build showed that 
Lite Mode: 99 tests ran
Full Mode: 577 tests ran

Reply:
https://github.com/OpenLiberty/open-liberty/wiki/FAT-Tests#repeating-fat-tests-for-jakarta-ee-9 : 

> It is recommended that only the latest version of a feature should be tested in LITE mode. Other versions should only be repeated in FULL mode.